compiler/accelerators/streamers/streamers.py

```diff
@@ -42,6 +108,7 @@ def size(self) -> int:
         """
         return len(self.streamers)
 
+    @deprecated("Please do not use this function, it is only valid in trivial cases")
     def temporal_dim(self) -> int:
         """
```

Comment:
These functions try to determine a single temporal dim for the streamers of several operands, but can only be done if they are all equal. This is often not the case.

Reply:
However, my mlperf dev branch still uses this, so I am not removing it just yet to make merging them more graceful.